roles/ipxe/tasks/main.yml

```diff
@@ -56,7 +56,7 @@
 
 - name: fetch the resulting files
   fetch: src={{ ansible_env.HOME }}/ipxe/src/bin/{{ item }} 
-         dest={{ ipxe_build_artifact_path }}/{{ item }} fail_on_missing=yes
+         dest={{ ipxe_build_artifact_path }}/monorail.ipxe fail_on_missing=yes
          flat=yes
   with_items:
     - ipxe.pxe
```

Comment:
with_items is the default build names,  I'm renaming them above.  Do you want to change the others to stay consistent?  Is this a -1?